packages/xen-api/examples/utils.mjs

```diff
@@ -20,7 +18,7 @@ exports.createInputStream = path => {
   return stream
 }
 
-exports.createOutputStream = path => {
+export function createOutputStream(path){
```

Comment:
Formatting is incorrect, avoid committing with `-n` as it prevents Prettier (the formater) from running.